pylib/gyp/generator/msvs.py

```diff
@@ -3786,10 +3812,10 @@ def _GenerateMSBuildProject(project, options, version, generator_flags):
         content += _GetMSBuildLocalProperties(project.msbuild_toolset)
     content += import_cpp_props_section
     content += import_masm_props_section
-    if spec.get("msvs_enable_marmasm"):
+    if spec.get("msvs_enable_marmasm") or True:
```

Comment:
@richard-townsend-arm I copied this from the downstream commit but it doesn't look right.

Reply:
LOL  :-)  That mod is not useful.  `if spec.get("msvs_enable_marmasm", True):` might be useful but it depends on how we want to deal with the case where `"msvs_enable_marmasm" not in spec`

Reply:
Yes, I think my `git commit -av` habit has struck again! It should compile fine without `or True`, and (if not) I'll fix that.

Reply:
A quick build on my own branch indicates that everything compiles fine without this `or True`. Feel free to remove it here and I can push a new patch to Node.js to remove it from there.

Reply:
Thank you for verifying! No need to push a patch to Node.js, we're going to downstream more changes from here.